### ISSUE TITLE
models auto detect crashes on @v0 

### ISSUE THREAD

**Issue by pustoshilov-d** [nanoporetech/dorado#550]
Hi
Running dorado 0.5.1 with 
`model=sup@v3.3,5mCG_5hmCG@v0`
returns
`[error] Failed to parse model argument. Version: 'v0.0.0' is not an available option`
However such model exists
```
Singularity> find . -name "*sup@v3.3*"
./dna_r9.4.1_e8_sup@v3.3
./dna_r9.4.1_e8_sup@v3.3_5mCG@v0.1
./dna_r9.4.1_e8_sup@v3.3_5mCG_5hmCG@v0
```

Guess there is problem with v0 parsing

**Comment by malton-ont** [comment]
Hi @pustoshilov-d. Thanks for reporting this. I can confirm this is a bug, and we'll work to get a fix into the next version. In the mean time, `@v0` will be auto-selected if you drop that specifier as that is the only `5mCG_5hmCG` available for `sup@v3.3`.
```
$ dorado basecaller sup@v3.3,5mCG_5hmCG ./data > calls.bam
[2024-01-02 09:43:21.704] [info]  - downloading dna_r9.4.1_e8_sup@v3.3 with httplib
[2024-01-02 09:43:23.266] [info]  - downloading dna_r9.4.1_e8_sup@v3.3_5mCG_5hmCG@v0 with httplib
...
```

**Comment by HalfPhoton** [comment]
This issue has been fixed. Thanks for bringing this to our attention.
